tests/page-objects/dashboard/DashboardPage.ts

```diff
@@ -114,6 +119,7 @@ class DashboardPage extends BasePage {
         selectItemsMobile: '.g-select-list_mobile',
         selectItemTitle: '.g-select-list__option',
         selectItemTitleDisabled: '.g-select-list__option_disabled',
+        chartkitControlSelect: slct('chartkit-control-select-items'),
```

Comment:
For this we can use `ControlQA.controlSelectItems` instead of a string. We try to use constants for such things.